web/client/translations/data.en-US.json

```
@@ -2728,6 +2735,100 @@
+                    },
+                    "navigation": {
+                        "next": {
+                            "title": "Navigation: Next",
+                            "text": "<p>Press this button to go to the next step. If the button is disabled, it means that either the data you configured is being validated, or it is invalid.</p>"
+                        },
+                        "back": {
+                            "title": "Navigation: Back",
+                            "text": "<p>Press this button to go to the previous step.</p>"
+                        },
+                        "close": {
+                            "title": "Navigation: Close",
+                            "text": "<p>Press this button to leave the context creation process without saving. This can be done at any point in the creation process.</p>"
+                        }
+                    },
+                    "fields": {
+                        "title": "General Settings",
+                        "text": "<p>The first creation step is to specify name of the context and title that should be set as the title of the window, when context is opened and active. Note, that context name can only contain latin letters, digits 0-9 and underscore character. Window title cannot be empty.</p>"
+                    },
+                    "showTutorial": {
+                        "title": "Show Tutorial",
+                        "text": "<p>If you wish to go through the tutorial for a step again, you can do it at any time by pressing this button.</p>"
+                    }
+                },
+                "configureMap": {
+                    "intro": {
+                        "title": "Configure Map",
+                        "text": "<p>In this step using MapStore map viewer tools you can configure a default map that will be opened upon accessing the context.</p>"
+                    },
+                    "reloadMap": {
+                        "title": "Reload Map Button",
+                        "text": "<p>If you wish to discard all the changes you've made to the map so far and start over, you can do so by pressing this button.</p>"
+                    }
+                },
+                "configurePlugins": {
+                    "intro": {
+                        "title": "Configure Plugins",
+                        "text": "<p>This step allows you to configure plugins that will be present in a context.</p>"
+                    },
+                    "availablePlugins": {
+                        "title": "Available Plugins",
+                        "text": "<p>This list contains MapStore plugins that can be added to a context. To select a plugin simply right click on it's card. Also, if you hold down your CTRL key, you can select multiple plugins.</p>"
```

```suggestion
                        "text": "<p>This list contains MapStore plugins that can be added to a context. To select a plugin simply click on it's card. Also, if you hold down your CTRL key, you can select multiple plugins.</p>"
```
Update the other translations